app/views/training_modules/index.html.slim

```diff
@@ -39,10 +39,10 @@ p.govuk-body Some modules also have short videos to enhance your understanding.
   .govuk-grid-column-full
     = govuk_accordion do |accordion|
       - @published_modules.map do |mod|
-        - accordion.section(heading_text: "Module #{mod.id}: #{mod.title}", summary_text: mod.description) do
-          .gem-c-govspeak= translate_markdown(mod.objective)
-          h2 What you&rsquo;ll learn 
-          p.govuk_body This module will cover:
+        - accordion.section(heading_text: "Module #{mod.id}: #{mod.title}", summary_text: mod.objective) do
+          p.govuk-body= mod.description
```

Comment:
Because `description` can now contain markdown, but in our YAML it doesn’t , it’s not clear that it can. 

You have one instance of it where markdown would get parsed ok and another here where it wouldn’t. So even looking at the template for an indication of what to expect would be misleading   I think.